docs/getting-started/my-account/README.md

```diff
@@ -122,32 +119,84 @@ Click the grey **Register device** button to register an authentication app. Ope
 
 Click the blue **Continue** button to finish the registration.
 
-![register authentication app](1572879640187.png)
+![openproject_my_account_authenticator_app](openproject_my_account_authenticator_app.png)
+
+## Access tokens
+To view and manage your OpenProject access tokens navigate to **My account** and choose **Access tokens** from the menu. 
+Access tokens allow you to grant external applications access to resources in OpenProject. 
+
+![openproject_my_account_access_tokens](openproject_my_account_access_tokens.png)
 
+### API
+API tokens allow third-party applications to communicate with this OpenProject instance via REST APIs. If you have not yet created an API token, this list will be empty. You may need admin privileges to be able to create an API.
```

Comment:
Maybe I would also add as a note the fact that right now you can only add a single API token.